App.js

```diff
@@ -16,9 +17,11 @@ const App = () => {
   return (
     <FlagsProvider flags={buildTimeFlags}>
       <MenuProvider>
-        <Theme use='default'>
-          <Entry />
-        </Theme>
+        <SafeAreaProvider>
```

Comment:
For some reasons the app works without this in the current develop branch, but that seems like a bug and was crashing the app locally for me.

Reply:
Wow. good catch!